hi-ml-azure/testazure/testazure/test_azure_util.py

```diff
@@ -2130,8 +2130,24 @@ def _mock_download_file(child_run_id: str, local_folder_child_run: Path, prefix:
     assert (local_download_folder / str(mock_run_1.id) / remote_file_path).exists()
 
 
-@patch("health_azure.utils.isinstance", return_value=True)
-def test_aggregate_hyperdrive_metrics(_: MagicMock) -> None:
+def test_get_metrics_for_hyperdrive_run() -> None:
+    # test the case where run id is provided
```

Comment:
Should we also test the case where a run object is provided, and where neither is provided?

Reply:
Done with a big refactoring of that test.